events/0216_seo.html

```diff
@@ -0,0 +1,168 @@
+                <div class="box">
+                    <h3 class="event__title box--small green">How to Build a Website: HTML &amp; CSS for Beginners</h3>
+                    <h3 class="event__title"><small>Saturday, 2/27 @ Alfa Jango</small></h3>
+                    <p><small>10am - 3pm &nbsp; | &nbsp; 1327 Jones Drive</small></p>
+                    <aside class="notes"></aside>
+                </div>
+            </section>
+
+            <section>
+                <h3>Upcoming GDI Event</h3>
+                <div class="box">
+                    <h3 class="event__title box--small green">Meet the Company Series: Pillar Technology</h3>
+                    <h3 class="event__title"><small>Monday, 2/29 @ The Forge by Pillar Technology</small></h3>
+                    <p><small>6:30pm - 8:30pm &nbsp; | &nbsp; 301 E. Liberty St, Suite 700</small></p>
+                    <aside class="notes"></aside>
+                </div>
+            </section>
+
+            <section>
+                <h3>Community Events</h3>
+                <ul class="box copy--xsmall left" style="width: 48%">
+                    <li><strong><span>Intro to Javascript (2-day Workshop)</span></strong><br /><small>Wednesday 2/10 &amp; Thursday 2/11 @ Grand Circus Detroit</small></li>
+                    <li><strong><span>Coffee House Coders Meetup</span></strong><br /><small>Wednesday 2/10 @ Espresso Royale - State Street</small></li>
+                    <li><strong><span>Hide Yo' Kids: Hacking Your Family's Connected Things</span></strong><br /><small>Thursday 2/18 @ Duo Security</small></li>
+                </ul>
+                <ul class="box copy--xsmall right" style="width: 48%">
+                    <li><strong><span>Rails &amp; ES6</span></strong><br /><small>Tuesday 2/23 @ Atomic Object</small></li>
+                    <li><strong><span>Intro to Search Engine Optimization</span></strong><br /><small>Saturday 2/27 @ Grand Circus Detroit</small></li>
+                    <li><strong><span>PHP Lightning Talks</span></strong><br /><small>Thursday 2/18 @ Nutshell</small></li>
+                    <li class="green"><em>Plus soooo many more...</em></li>
+                </ul>
+                <p class="clear copy--small"><a href="http://bit.ly/gdiaa-community">bit.ly/gdiaa-community</a></p>
+                <aside class="notes"></aside>
+            </section>
+
+            <!-- Sponsors -->
+            <section>
+                <h3>Sponsors!</h3>
+                <img src="images/sponsors-2015.3.png" class="img--bare box--small" />
+            </section>
+
+            <!-- Sponsor Spotlight -->
+            <section>
+                <h3>Sponsor Spotlight!</h3>
+                <img src="images/atomic_object.png" class="img--bare box" width="500" />
+            </section>
+
+            <section>
+                <h3>Sponsor Spotlight!</h3>
+                <img src="images/stone-logo.png" class="img--bare box" width="200" />
+            </section>
+
```

Comment:
Can you add a slide between these to that just contains the link to the actual workshop slides?